src/sentry/static/sentry/less/shared-components.less

```diff
@@ -1484,28 +1484,6 @@ ul.radio-inputs {
   }
 }
 
-/**
- * Truncate component.
- */
-.truncated {
```

Comment:
Did you check if we use this in `getsentry`?

Reply:
Can confirm that `getsentry` does not use this anywhere.